src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/ECAlgorithm.cs

```diff
@@ -862,5 +862,87 @@ public override void ImportFromEncryptedPem(ReadOnlySpan<char> input, ReadOnlySp
             // override remains for compatibility.
             base.ImportFromEncryptedPem(input, passwordBytes);
         }
+
+        /// <summary>
+        /// Exports the current key in the ECPrivateKey format, PEM encoded.
+        /// </summary>
+        /// <returns>A string containing the PEM-encoded ECPrivateKey.</returns>
+        /// <exception cref="CryptographicException">
+        /// The key could not be exported.
+        /// </exception>
+        /// <remarks>
+        /// <p>
+        ///   A PEM-encoded ECPrivateKey will begin with <c>-----BEGIN EC PRIVATE KEY-----</c>
+        ///   and end with <c>-----END EC PRIVATE KEY-----</c>, with the base64 encoded DER
+        ///   contents of the key between the PEM boundaries.
+        /// </p>
+        /// <p>
+        ///   The PEM is encoded according to the IETF RFC 7468 &quot;strict&quot;
+        ///   encoding rules.
+        /// </p>
+        /// </remarks>
+        public unsafe string ExportECPrivateKeyPem()
+        {
+            byte[] exported = ExportECPrivateKey();
+
+            // Fixed to prevent GC moves.
+            fixed (byte* pExported = exported)
```

Comment:
If the key was already exported above, this is too late to guarantee no copies have been made.

Reply:
That is true but it's a pattern that occurs quite frequently in other places as a "best effort". Two examples: 

* https://github.com/dotnet/runtime/blob/6b6310b1c96457a2950c526dd1e46ed055343de2/src/libraries/Common/src/System/Security/Cryptography/CngPkcs8.cs#L412
* https://github.com/dotnet/runtime/blob/899bf9704693661d6fe53fdb7f737f76447efa14/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/ECDsa.cs#L1033

But there are many more.